bagger/README.md

```diff
@@ -0,0 +1,229 @@
+# ReBACH-Bagger
+
+This tool implements the "last mile" of ReDATA's preservation strategy by
+ingesting a data/metadata package, generating a preservation-ready bag using
+the APTrust DART tool, and uploading the completed bag to offsite storage.
```

Comment:
Would a brief reference to a higher-level summary of ReDATA's preservation strategy / workflow be warranted there? Something as short as "For more details on ReDATA's workflow, see https://url-goes-here". This assumes such a resource exists...

Reply:
I suggest to add APTrust DART's GitHub link (https://github.com/APTrust/dart), which provides the DART code and  DART documentation as well as BagIt specification.

Reply:
@jcoliver Alas, such a thing doesn't exist publicly. The closest thing is the diagram in the spec. I've added a mention of that. @yhan818 good idea on the link.